src/app/components/CreateButton.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import { useState } from 'react';
+
+const buttonClassnames: Record<string, string> = {
+  button:
+    'relative right-1 w-16 h-16 bg-white active:bg-cyan-300 border-2 text-teal border-teal rounded-full',
+  subButton:
+    'p-2 bg-white active:bg-cyan-300 border-2 text-teal border-teal rounded-full',
+};
+
+function showSubCreateButtons(): JSX.Element {
+  return (
+    <div className="flex flex-col gap-4">
+      <button className={buttonClassnames.subButton}>
+        <span className="text-xl">Juz</span>
+      </button>
+      <button className={buttonClassnames.subButton}>
+        <span className="text-xl">Ayah</span>
+      </button>
+      <button className={buttonClassnames.subButton}>
+        <span className="text-xl">Surah</span>
+      </button>
+    </div>
+  );
+}
+
+export const CreateButton = (): JSX.Element => {
+  const [isShowSubButtons, setIsShowSubButtons]: [
+    boolean,
+    Dispatch<SetStateAction<boolean>>,
+  ] = useState<boolean>(false);
```

Comment:
this typing definition is sufficient
```
const [isShowSubButtons, setIsShowSubButtons] = useState<boolean>(false);
```

Reply:
true, but currently we using the eslint rule that will raise error when we don't specify type of the variables